### DISCUSSION TITLE
Loading gpt4chan_model_float16... Can't determine model type from model name.

### DISCUSSION THREAD

**Discussion by daveckw** [oobabooga/text-generation-webui#958] [General]
Loading gpt4chan_model_float16... Can't determine model type from model name. 
Please specify it manually using --model_type argument

Hi, may I know how to fix the issue above?
Thank you

**Comment by MaiJames** [comment]
To run the model, place the model files on `models/gpt4chan_model`.
Then download all the small files (all the `.json` files) from: https://huggingface.co/EleutherAI/gpt-j-6b/tree/main and place them inside the same `models/gpt4chan_model` folder.

Then edit your `.bat` file arguments with:

`call python server.py --auto-devices --chat --model gpt4chan_model`

That's what worked for me.

By the way, I guess the model gets the `model_type` attribute from the `config.json` downloaded from the GPT-J-6B repo, and inside the file it says `"model_type": "gptj"`

I hope it helps.

**Comment by oobabooga** [comment]
You are trying to load a 16-bit model in 4-bit mode with 4-bit command-line arguments.
